traffic_ops/traffic_ops_golang/invalidationjobs/invalidationjobs.go

```diff
@@ -270,6 +475,12 @@ func (job *InvalidationJob) Read(h http.Header, useIMS bool) ([]interface{}, err
 			return nil, nil, fmt.Errorf("parsing db response: %v", err), http.StatusInternalServerError, nil
 		}
 
+		// NamedQuery (nor prepare statements) cannot handle string functions in SELECT such as:
+		// 	CONCAT('TTL:', ttl_hr, 'h') AS parameters
```

Comment:
I'm guessing this might be because of the `:` in `TTL:`. I think using `TTL::` might work? That is to say, I think it might actually be possible to use `CONCAT` and other string functions in `NamedQuery`.